workspace.json

```diff
@@ -0,0 +1,11 @@
+{
+  "version": 2,
+  "projects": {
+    "akita-read": "packages/akita-read",
+    "angular-logger": "packages/angular-logger",
+    "angular-teleport": "packages/angular-teleport",
+    "concurrency-utils": "packages/concurrency-utils",
+    "logger": "packages/logger",
+    "rxjs-read": "packages/rxjs-read"
+  }
+}
```

Comment:
This file should not be needed since this is a yarn project.

Reply:
https://github.com/cloudflightio/cloudflight-js-utils/actions/runs/2979647474

I did not add it initially but then Nx was not able to identify the dependency correctly and the build failed